Makefile

```diff
@@ -325,7 +379,9 @@ clean:
 	cd Firefox && $(MAKE) clean
 	cd Webmin && $(MAKE) clean
 	cd Chromium && $(MAKE) clean
-	rm -f scap-security-guide.spec
+	rm -f $(RPM_SPEC)
```

Comment:
Hi @pthierry38, thanks for the comment. I looked at `dh_make` but it seemed a bit overkill just to package some xml/html files.
Will test your `dh_make` line and see if I can reduce the lines of code.